src/applications/vaos/components/VAFacilityLocation.jsx

```diff
@@ -40,11 +41,12 @@ export default function VAFacilityLocation({
     );
   }
 
+  const name = clinicName || facilityName || facility?.name;
+
   return (
     <>
-      <h4 className="vaos-appts__block-label">
-        {clinicName || facilityName || facility?.name}
-      </h4>
+      {!isV2 && <h4 className="vaos-appts__block-label">{name}</h4>}
+      {isV2 && name}
```

Comment:
New design of facility location is identical, but the facility name is not a header or bolded.